airflow/www/static/css/main.css

```diff
@@ -465,6 +465,10 @@ label[for="timezone-other"],
   z-index: 1070;
 }
 
+details summary {
+  display: list-item;
+}
+
 .menu-scroll {
   max-height: 300px;
```

Comment:
I'm no css expert but the values should be in 'em' not 'px' so the display can be dynamically based off the screen size of the viewer.

Reply:
Maybe you should provide and test PR with a fix @tronlightracer ? This is the best way to become one of > 2100 contributors to Airflow (mostly people like you).